kalite/distributed/static/js/distributed/exercises/views.js

```diff
@@ -436,8 +436,7 @@ window.ExerciseView = Backbone.View.extend({
 
 });
 
-
-window.ExercisePracticeView = Backbone.View.extend({
+window.ExerciseWrapperBaseView = BaseView.extend({
```

Comment:
Can you please add comments explaining the intended use? This appears to be an abstract base class.